data/gallery.js

```diff
@@ -283,27 +283,32 @@ function displayHash() {
 
 const forwardButtClick = () => {
   const forwardBtn = document.getElementsByClassName('forwardButt');
-  forwardBtn[0].addEventListener('click', () => {
-    if (
-      currentImageIndex <
-      galleryAlbums[currentAlbumIndex].fields.images.length - 1
-    ) {
-      currentImageIndex += 1;
-      updateSingleImage();
-    }
-  });
+  forwardBtn[0].addEventListener('click', nextImage);
 };
+function nextImage() {
+  if (
+    currentImageIndex <
+    galleryAlbums[currentAlbumIndex].fields.images.length - 1
+  ) {
+    currentImageIndex += 1;
+    console.log(currentImageIndex);
+    updateSingleImage();
+  }
+}
 
```

Comment:
noice.  Optional update to the code if you want to make it slightly more succinct:

you can merge your  'previousImage' and 'nextImage' functions and pass in a parameter with the direction, e.g.

```
function changeImage(direction) {
  if (
    direction === 'next' &&
    currentImageIndex < galleryAlbums[currentAlbumIndex].fields.images.length - 1) {
    currentImageIndex += 1;
    updateSingleImage();
  }
  else if (direction === 'previous' && currentImageIndex > 0) {
    currentImageIndex -= 1;
    updateSingleImage();
  }
}
```